src/kicker/main.c

```diff
@@ -366,6 +382,7 @@ ISR(TIMER0_COMPA_vect) {
         // stop prescaled timer
         TCCR0B &= ~_BV(CS01);
     }
+    cli();
```

Comment:
This disables interupts. Has this code been workign????

Reply:
That was a temporary (and broken) commit that I was working on, I'll let you know when this is ready for a merge.